src/index.jsx

```diff
@@ -44,20 +115,31 @@ class Container extends Component {
           zoomMin={scale.zoomMin}
           zoomMax={scale.zoomMax}
         />
-        <div className={`layout ${isOpen ? 'is-open' : ''}`}>
-          <div className="layout__side">
+        <div className={`layout ${isOpen ? 'is-open' : ''}`} ref={(layout) => { this.layout = layout }}>
+          <div className="layout__side" ref={(sidebar) => { this.sidebar = sidebar }}>
             <Sidebar
               timebar={timebar}
               tracks={tracks}
               toggleTrackOpen={toggleTrackOpen}
+              isHeaderSticky={isHeaderSticky}
+              headerHeight={headerHeight}
+              width={sidebarWidth}
             />
           </div>
-          <div className="layout__main">
+          <div className="layout__main" ref={(layoutMain) => { this.layoutMain = layoutMain }}>
             <Timeline
               now={now}
               time={time}
               timebar={timebar}
               tracks={tracks}
+              getMarkerOffset={this.getMarkerOffset}
+              getHeaderHeight={this.getHeaderHeight}
+              getTimelineWidth={this.getTimelineWidth}
+              isHeaderSticky={isHeaderSticky}
+              stickyHeader={stickyHeader}
+              headerHeight={headerHeight}
+              timelineVisualWidth={timelineVisualWidth}
+              isOpen={isOpen}
```

Comment:
A lot of top-level properties added just for "sticky header" functionality. I'd tuck these away under a `stickyHeader` prop e.g.

```
const stickyHeader = { updateMarkerOffset, updateHeaderHeight }
<Timeline now={now} stickyHeader={stickyHeader} ... />
```